GOVERNANCE.md

```diff
@@ -0,0 +1,69 @@
+# Cloud Native Buildpacks Governance
+This document defines the governance structure for Cloud Native Buildpacks. This project is committed to building an open, inclusive, productive and self-governing open source community that maintains high-quality tools for translating source code into OCI images. The community is governed by this document with the goal of defining how said community should work together to achieve this goal.
+
+## Repositories
+The following code repositories are governed by the Cloud Native Buildpacks community and maintained under the [buildpacks](https://github.com/buildpacks) organization.
+
+* [spec](https://github.com/buildpacks/spec)
+* [lifecycle](https://github.com/buildpacks/lifecycle)
+* [pack](https://github.com/buildpacks/pack)
+* [docs](https://github.com/buildpacks/docs)
+* [rfcs](https://github.com/buildpacks/rfcs)
+* [resources](https://github.com/buildpacks/resources)
+
+## Core Team
+The Core team is responsible for the direction of the project (roadmap), subteam leadership, the spec, and cross-cutting concerns. This includes:
+
+* [spec](https://github.com/buildpacks/spec)
+* [rfcs](https://github.com/buildpacks/rfcs)
+
+Core team members are responsible for voting on RFCs and other changes to the project such as changes in leadership or community roles. No company will have a combined voting block of larger than 50%.
+## Sub-teams
+Sub-teams are responsible for narrower sets of concerns related to specific aspects of the project. Each sub-team will include at least one core team member to help align with the broader roadmap.
+### Roles
+
+#### Maintainers
+Maintainers are in charge of the day to day maintenance of the team's projects. They review
+Reviews PRs, ability to merge.
+Ensures code 	quality
+
+New maintainers must already be contributors, must be nominated by an existing maintainer or core team member, and must be elected by a supermajority of the core team. Likewise, maintainers can be removed by a supermajority of the core team or can resign by notifying one of the maintainers.
+
+#### Contributors
+Contributors are those who make regular contributions to the project (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). New contributors may be self-nominated or be nominated by existing contributors, and must be elected by a supermajority of that project’s maintainers.
+
+### Implementation Team
+The Implementation team is responsible for maintaining the components that constitute the Cloud Native Buildpacks reference implementation of the specification. This includes:
+
+* [lifecycle](https://github.com/buildpacks/lifecycle)
+* [libbuildpack](https://github.com/buildpacks/libbuildpack)
```

Comment:
We should probably group `imgutil` with the Implementation team, because it is imported by `lifecycle` and its performance and accuracy are essential to the behavior of `lifecycle`. 

It's current permissions don't seem to fit neatly into any bucket- https://github.com/buildpacks/imgutil/settings/access.